### ISSUE TITLE
View selection drop down menu is duplicated on each party section

### ISSUE THREAD

**Issue by GoogleCodeExporter** [halalaninitiative/halalan#75]
```
What steps will reproduce the problem?
1. Go to Candidates tab
2. Under View drop down menu, select Default Election

What is the expected output? What do you see instead?
Notice that the View selection drop down menu is duplicated on each party 
section. It gives the impression that you can have different views for each 
party section. Can we just keep this view selection on top?

What version of the product are you using? On what operating system?
Halalan 1.5.0
```

Original issue reported on code.google.com by `daru...@gmail.com` on 6 Jan 2011 at 11:44

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision b67a1416ca.
```

Original comment by `daru...@gmail.com` on 8 Jan 2011 at 3:52
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
Original comment by `waldemar...@gmail.com` on 7 Feb 2011 at 9:14
- Changed state: **Verified**
